static/app/views/performance/landing/index.tsx

```diff
@@ -212,6 +215,14 @@ export function PerformanceLanding(props: Props) {
                       router={props.router}
                       {...metricsDataSide}
                     />
+                    <DynamicSamplingAlert
+                      organization={organization}
+                      selectProject={
+                        selectedProjects.length === 1
+                          ? projects.find(project => project.slug === selectedProjects[0])
+                          : undefined
```

Comment:
if we click on "All Projects" in the project selector, the length of `selectedProjects` will be 1 ([-1])

but I guess it's ok in this case because `projects.find` will return undefined

maybe something we could explicitly handle in the logic or at least leave a comment for future reference